packages/ibm-products/src/components/TagSet/TagSet.mdx

```diff
@@ -6,6 +6,10 @@ import * as stories from './TagSet.stories';
 # TagSet
 
 [TagSet usage guidelines](https://pages.github.ibm.com/carbon/ibm-products/components/tag-set/usage/)
+|
+[Carbon Tag usage guidelines](https://carbondesignsystem.com/components/tag/usage/)
+|
+[Carbon Tag documentation](https://react.carbondesignsystem.com/?path=/docs/components-tag--overview)
```

Comment:
Doesn’t look like these updates are applied since TagSet doesn’t use the MDX file at the moment. (I see it but it’s commented out.)

Reply:
The import of the `.mdx` file was commented out in the `.stories.js` file.

If you and I were part of the "in crowd", we would have known that because the `page` property wasn't defined, then _by default_ the Storybook setup was automatically injecting the following...

```
export default {
  page: () => (
    <StoryDocsPage />
  ),
}
```

...which is why we didn't understand why our `.mdx` updates weren't being applied.

Whoever did the last update didn't clean up behind themselves, leaving us to deal with the mess.